### PR TITLE
Fix Cube Mapping & Minor Tweaks

### PR DESCRIPTION
**Pull request by LukeUsher**
image

### CODE REVIEW

File: src/Common/EmuEEPROM.cpp [hunk 1]
```diff
@@ -187,6 +187,7 @@ xboxkrnl::XBOX_EEPROM *CxbxRestoreEEPROM(char *szFilePath_EEPROM_bin)
 		pEEPROM->UserSettings.ParentalControlMovies = XC_PC_MAX; // = XC_PRTL_CRTL_MAX
 		pEEPROM->UserSettings.MiscFlags = 0;  // No automatic power down
 		pEEPROM->FactorySettings.AVRegion = AV_STANDARD_NTSC_M | AV_FLAGS_60Hz;
+		memcpy(&pEEPROM->FactorySettings.SerialNumber, "Cxbx-R      ", 12);
 		pEEPROM->EncryptedSettings.GameRegion = XC_GAME_REGION_NA;
```
Comment (ergo720) [inline src/Common/EmuEEPROM.cpp]: I would suggest to also add the above in DlgEepromConfig.cpp (look at IDC_EE_RESET). If you don't, then the reset functionality of the eeprom menu will still reset the serial number to a null value.

Comment (LukeUsher) [reply src/Common/EmuEEPROM.cpp]: Thanks! That begs the question of why eeprom data is reset in different places...  I'll clean it up later.

**Comment by LukeUsher** [comment]
Appveyor failed to build the debug build, don't merge until this is solved.

**Comment by Margen67** [comment]
The cubemap/specular on the car in RaceX is still missing.

**Comment by LukeUsher** [comment]
@Margen67 probably a different issue not strictly related to this one.

**Comment by LukeUsher** [comment]
Merging: It may be a little premature but this is such a small/simple fix in reality that it's minimal impact.

**Comment by Margen67** [comment]
Crash Bandicoot: TWoC no longer gets past the Universal logo (confirmed regression):
```---------------------------
Cxbx-Reloaded
---------------------------
Received Fatal Message:

* CreateCubeTexture Failed!

Error: 
Desc: 

---------------------------
OK   
---------------------------
```

**Pull request merged**